core/UrlHelper.php

```diff
@@ -135,10 +143,20 @@ public static function isLookLikeUrl($url)
     {
         return preg_match('~^(([[:alpha:]][[:alnum:]+.-]*)?:)?//(.*)$~D', $url, $matches) !== 0
             && strlen($matches[3]) > 0
-            && !in_array($matches[1], array('javascript:', 'vbscript:', 'data:'))
+            && !preg_match('/^(javascript:|vbscript:|data:)/i', $matches[1])
             ;
     }
 
+    public static function isLookLikeSafeUrl($url)
+    {
+        if (strpos($url, ':') === false) {
```

Comment:
Looks good to merge but maybe we should here also execute `self:: isLookLikeUrl ($url)` and only return true if it actually is a url?

Reply:
`tel:555...` and `sms:...` links do not pass isLookLikeUrl, since they don't have `//`. I didn't want to regress anything since isLookLikeUrl is used in tracking and probably everywhere else, so didn't modify it.

Reply:
Note: I did try and make the `//` optional, but then everything passes `isLookLikeUrl`